web_services/index.ipynb

```diff
@@ -105,6 +105,163 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**How to use the QGIS Temporal Controller with WMS-T layers**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. To use the **Temporal Controller** for WMS-T layers, ensure that in the Temporal section of the layer's **Layer Properties**, the **Dynamic Temporal Control** is selected. You can select the time range to be **Automatic** or **Fixed time range**. \n",
+    "For the **Automatic** setting, the temporal handling is delegated to the data provider and time based filtering is applied to the features of the layer such that the features will be visualized for the animation range set for the **Animated temporal navigation**, or the range set for the **Fixed range temporal navigation**. \n",
+    "For the **Fixed time range** setting the user manually sets the temporal range in which all the features of the layer will be visualised only if the animation range, for the Animated temporal navigation, or the Range, for the Fixed range temporal navigation, is within this temporal range. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_10.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_11.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "2. On the **Map Navigation Toolbar** select the clock symbol for the **Temporal Controller Panel** to activate the **Temporal Controller Panel**. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_12.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_13.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "3. To turn on the Fixed range temporal navigation, select the **Fixed range temporal navigation** icon ![Temporal Navigation Fixed Range](../_static/web_services/mTemporalNavigationFixedRange.svg) on the **Temporal Controller Panel** and set the time range for the layer features to be visualized. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_14.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "4. To turn on the Animated temporal navigation, select the **Animated temporal navigation** icon ![Temporal Navigation Animated](../_static/web_services/mTemporalNavigationAnimated.svg) on the **Temporal Controller Panel**, and set animation range and the time step for the animation.  \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_15.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_16.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n",
+    "\n",
+    "5. Click on the play icon ![play](../_static/web_services/forward.svg) to view the animation. \n",
```

Comment:
We need to explain also about the use of other navigation buttons apart for the play button.